app/views/shared/_navbar_user_messages_counter.html.slim

```diff
@@ -1,5 +1,5 @@
 - if current_user.messages.unreaded.count > 0
   .badge.is-badge-small.is-badge-outlined[data-badge="#{current_user.messages.unreaded.count}"]
-    = current_user.full_name
+    = current_user.full_name.truncate(15)

```

Comment:
use helper, truncate

Reply:
and never use this approach in views

Reply:
i use it

Reply:
oh or you mean rails helper becose i use ruby method